scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1725,8 +1671,6 @@ void BPF_STRUCT_OPS(layered_dispatch, s32 cpu, struct task_struct *prev)
 		return;
 	}
 
-	u32 my_llc_id = cpu_to_llc_id(cpu);
```

Comment:
Here, I think cpu and cctx are pointing to the same cpu. It's actually confusing that we're doing `lookup_cpu_ctx(-1)` instead of `lookup_cpu_ctx(cpu)`. I'll comment on the part which was buggy.